src/celldega/pre/boundary_tile.py

```diff
@@ -272,6 +324,8 @@ def make_cell_boundary_tiles(
         path_transformation_matrix, header=None, sep=" "
     ).values
 
+    gene_str_to_int_mapping, cell_str_to_int_mapping = _get_name_mapping(path_transformation_matrix.replace('/micron_to_image_transform.csv',''))
```

Comment:
I feel like this is a little inefficient in that its making the gene map here even though its not needed in this function. We could make an argument to tell `_get_name_mapping` which mapping we want: gene or cell.

Reply:
fixed.